.vscode/deno_resolve_npm_imports.json

```diff
@@ -8,6 +8,9 @@
   ],
   "imports": {
     "@remix-run/server-runtime": "https://esm.sh/@remix-run/server-runtime@nightly",
+    "@remix-run/deno": "../packages/remix-deno/index.ts",
+    "@remix-run/deno/": "../packages/remix-deno/",
```

Comment:
Since we already have `@remix-run/deno` defined, I think we can remove this line?

```suggestion
```

Reply:
No, that one catches subpaths

Reply:
What @ascorbic said.